int32 num_poste
int32 num_produit
int32 duree
